tests/main/snap-discard-ns/mount.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+if [ "$(command -v python3)" != "" ]; then
+	exec python3 ./mount.py "$@";
+elif [ "$(command -v python2)" != "" ]; then
+	exec python2 ./mount.py "$@";
```

Comment:
Why this logic if it works with both versions of python, why not simply check for `/usr/bin/python` (and have `#!/usr/bin/env python` shebang?

Reply:
`python` is usually `python2` but I wanted to be explicit. In practice we will only pick up the other case (python2) on the Amazon Linux 2 systems.

Reply:
Fair enough, not a problem at all, just looks a little elaborate considering the fact that we don't actually care which python version it is.